iocore/net/ALPNSupport.cc

```diff
@@ -144,3 +144,36 @@ ALPNSupport::registerNextProtocolSet(SSLNextProtocolSet *s, const SessionProtoco
   this->npnSet       = s;
   npnSet->create_npn_advertisement(protoenabled, &npn, &npnsz);
 }
+
+bool
+ALPNSupport::process_alpn_protocols(std::string_view protocols, unsigned char *client_alpn_protocols, int &alpn_array_len)
```

Comment:
It would be nice if you could put this into some module that parses setting values. Maybe RecHttp.cc, which parses server_ports.

Reply:
RecHttp.cc works well. I put it there and added unit tests for it as well.